index.html

```diff
@@ -269,6 +272,45 @@ <h3>
       The <dfn data-dfn-for="DigitalCredential">data</dfn> member is the
       credential's response data.
     </p>
+    <h3>
+      [[\Store]](credential, sameOriginWithAncestors) internal method
+    </h3>
+    <p>
+      When invoked, the <dfn class="export" data-dfn-for=
+      "DigitalCredential">[[\Store]](credential, sameOriginWithAncestors)</dfn>
+      MUST call the default implementation of {{Credential}}'s
+      {{Credential/[[Store]](credential, sameOriginWithAncestors)}} internal
+      method with the same arguments.
+    </p>
+    <h3>
+      [[\Create]](origin, options, sameOriginWithAncestors) internal method
+    </h3>
+    <p>
+      When invoked, the <dfn class="export" data-dfn-for=
+      "DigitalCredential">[[\Create]](origin, options,
+      sameOriginWithAncestors)</dfn> internal method MUST run the following
+      steps:
+    </p>
+    <ol class="algorithm">
+      <li>Return `null`.
```

Comment:
Can't we also call the default implementation of `Create` here, as you did to `Store`?

The default implementation seems to return `null` too?

https://www.w3.org/TR/credential-management-1/#algorithm-create-cred

Reply:
Yes, you are right. We can call directly into the default one.